### PR TITLE
Adding TCP support

### PR DESCRIPTION
**Pull request by msmakouz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -620,6 +620,91 @@ protected const LOAD = [
 php app.php queue:pause local
 ```
 
+### TCP
```
Comment (butschster) [inline README.md:623]: Please add information about bootloader

**Review by butschster — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -620,6 +620,107 @@ protected const LOAD = [
...
+     * Services for each server.
+     */
+    'services' => [
+        'tcp_access_point_1' => SomeService::class,
+        'server2' => OtherService::class
+    ],
+
+    /**
+     * Interceptors, this section is optional.
+     */
```
Comment (butschster) [inline README.md:683]: Add link to interceptors 
https://spiral.dev/docs/cookbook-domain-core/2.8/en#core-interceptors

File: README.md [hunk 3]
```diff
@@ -620,6 +620,107 @@ protected const LOAD = [
...
+    'interceptors' => [
+       'tcp_access_point_1' => [SomeInterceptor::class, OtherInterceptor::class], // several interceptors
+       'server2' => SomeInterceptor::class // one interceptor
+    ],
+    
+    'debug' => true
```
Comment (butschster) [inline README.md:689]: Add `env('TCP_DEBUG', false)`

File: README.md [hunk 4]
```diff
@@ -620,6 +620,107 @@ protected const LOAD = [
...
+    'services' => [
+        'tcp_access_point_1' => SomeService::class,
+        'server2' => OtherService::class
+    ],
+
+    /**
+     * Interceptors, this section is optional.
+     */
+    'interceptors' => [
+       'tcp_access_point_1' => [SomeInterceptor::class, OtherInterceptor::class], // several interceptors
```
Comment (butschster) [inline README.md:685]: Split into lines

**Review by butschster — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (msmakouz) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (msmakouz) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (msmakouz) [reply README.md:684]: Added

**Review by butschster — APPROVED** [review]

**Pull request merged**